bench/workgen/workgen.cxx

```diff
@@ -532,45 +568,49 @@ int Monitor::run() {
                 (f) << "}";                                                \
             } while(0)
 
-            (*_json) << "{";
-            if (first) {
-                (*_json) << "\"version\":\"" << version << "\",";
-                first = false;
-            }
-            (*_json) << "\"localTime\":\"" << time_buf
-                     << "\",\"workgen\":{";
-            TRACK_JSON(*_json, "read", interval.read, percentiles, "");
-            (*_json) << ",";
-            TRACK_JSON(*_json, "insert", interval.insert, percentiles, "");
-            (*_json) << ",";
-            TRACK_JSON(*_json, "update", interval.update, percentiles, "");
-            (*_json) << ",";
-            TRACK_JSON(*_json, "checkpoint", interval.checkpoint, percentiles,
-              "\"active\":" << (checkpointing ? "1," : "0,"));
-            (*_json) << "}}" << std::endl;
-        }
-
-        uint64_t read_max = interval.read.max_latency;
-        uint64_t insert_max = interval.insert.max_latency;
-        uint64_t update_max = interval.update.max_latency;
-
-        if (read_max > latency_max)
```

Comment:
Those traces are read by a Jenkins job now in order to detect too long latencies using the pattern:
`max latency exceeded.*read|insert|update`